### ISSUE TITLE
Fix admin reminder email text

### ISSUE THREAD

**Issue by mokolodi1** [mokolodi1/aesop-afghanistan#39]
I got an email that reads:
```
Hi AESOP Phone Buddy Admin,

Don't forget to fill out the Process sheet for the AESOP phone buddies!

<function DatabaseConnector.get_database_link at 0x7ff1a2a905e0>

Best,

Teo
```

Goal is to fix the code such that a link to the spreadsheet is sent instead of `<function DatabaseConnector.get_database_link at 0x7ff1a2a905e0>`.

**Comment by mokolodi1** [comment]
Created branch to work on: https://github.com/mokolodi1/aesop-afghanistan/tree/fix-39

**Comment by mokolodi1** [comment]
Fixed as part of #54
